mousetrap.js

```diff
@@ -1018,4 +1023,4 @@
             return Mousetrap;
         });
     }
-}) (window, document);
+}) (typeof window !== 'undefined' ? window : null, typeof document !== 'undefined' ? document : null);
```

Comment:
Is there ever a case document would be undefined, but window wouldn't be? If so I think that would cause an error now. If it is going to return early when window is not there then probably there is no need to check `typeof document`